sway-core/src/metadata.rs

```diff
@@ -14,7 +14,7 @@ use std::{collections::HashMap, path::PathBuf, sync::Arc};
 ///! indices repeatedly.
 
 #[derive(Default)]
-pub(crate) struct MetadataManager {
+pub struct MetadataManager {
```

Comment:
Why does this need to leak out of the crate?

Reply:
Without this, I get `can't leak crate-private type` in `compile.rs`.